### ISSUE TITLE
Guide not working correctly

### ISSUE THREAD

**Issue by thedesignatedgeek** [djp952/pvr.hdhomerundvr#62]
Issue:
Guide does not show all channels. (example: Missing at lease 18.1, 18.2, 18.3, 18.4)
Channels does show all channels, but "current" show on some channels is wrong.
Timer rules seem to work correctly, even when channels are not accessable from guide.
This started about 5 or 6 days ago.
On DVR drive, log files started showing up about the same time.
Updated to current version yesterday, no changes
Current version raspbian-armhf-leia-2.2.2.181
Please see:
Kodi.log
HdHomerun data.txt
DVR-20190909.log

Thank you in advance,
TheDesignatedGeek

[kodi.log](https://github.com/djp952/pvr.hdhomerundvr/files/3590298/kodi.log)
[HDHomerun data.txt](https://github.com/djp952/pvr.hdhomerundvr/files/3590305/HDHomerun.data.txt)
[DVR-20190909.log](https://github.com/djp952/pvr.hdhomerundvr/files/3590308/DVR-20190909.log)

**Comment by djp952** [comment]
Hi! Kodi has problems when channels are added/removed, I ran into this recently as well.  The easiest solution is to go into the Kodi PVR & Live TV settings and choose 'Clear Data'.  This will reset all of the internal Kodi stuff and should clear it up. You won't lose any data but the EPG will need to fully reload.

I submitted a couple changes to Kodi recently (18.3) that fixed some of this but clearly there is more going on. If the channels are in Channels then the PVR is OK. I will find some time to try to help Kodi fix this.

Let me know if this helps!
